src/proxy.conf.js

```diff
@@ -1,6 +1,7 @@
 require('dotenv').config();
 
-const API_HOST_CONTENT = process.env.API_HOST_CONTENT || 'http://0.0.0.0:8080';
+const API_HOST_CONTENT =
+  process.env.API_HOST_CONTENT || 'http://adfdev.envalfresco.com';
```

Comment:
I don't think default should be our internal env. Please change back to localhost